pom.xml

```diff
@@ -64,6 +64,21 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.dataformat</groupId>
+        <artifactId>jackson-dataformat-yaml</artifactId>
+        <version>${jackson.version}</version>
```

Comment:
Version can/should be removed, should come from `jackson-bom` (I'll remove)

Reply:
Oh. This is in wrong section -- `<dependencyManagement>` is for versions, not dependencies: need to move to under `<dependencies>`. Will change.